docs/sphinx_ext/self_service_error_codes_extension.py

```diff
@@ -1,216 +0,0 @@
-# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-# SPDX-License-Identifier: Apache-2.0
-
-from collections import defaultdict
```

Comment:
FYI @fabiotudone-da Removing this Sphinx plugin. Replacing it with `ErrorCodeInventoryDocsGenApp.scala`